sdk/nodejs/policy/policy.ts

```diff
@@ -797,10 +797,25 @@ export function validateStackResourcesOfType<TResource extends Resource>(
     return fn;
 }
 
+/**
+ * The set of arguments used to report a policy violation.
+ */
+export interface ReportViolationArgs {
+    urn: string;
+    name: string;
+    description: string;
+    enforcementLevel?: EnforcementLevel;
+    severity?: Severity;
```

Comment:
Let's remove this for now. I'll add it back after this PR when I plumb severity through over gRPC (working on that now).